src/SDKs/OperationalInsights/Management/OperationalInsights.Test/OperationalInsights.Test.csproj

```diff
@@ -15,7 +15,7 @@
     
     <ProjectReference Include="..\Management.OperationalInsights\Microsoft.Azure.Management.OperationalInsights.csproj" />  
   </ItemGroup>
-  
+
```

Comment:
@csuich2 please use latest version of Resource manager to record all your tests.
You are currently using 1.1.0 version and the latest version is 1.6.0
Simply add reference to the 1.6.0 and use the proper using statements to use the new client from the new nuget package.
Do not try to uninstall 1.1.0. 1.1.0 and 1.6.0 are side by side compatible.

Reply:
This is in the 'management' SDK which my team doesn't own. I think this is only showing up in the PR because I moved the file and I opened it in VS to compare, which I believe is why the line at the end was added. I'll undo this change as well as the addition of the AssemblyInfo.cs since I don't own this SDK.

Reply:
This is in the 'management' SDK which my team doesn't own. I think this is only showing up in the PR because I moved the file and I opened it in VS to compare, which I believe is why the line at the end was added. I'll undo this change as well as the addition of the AssemblyInfo.cs since I don't own this SDK.

Reply:
Actually, it turns out git just thinks I moved the old Management AssemblyInfo to DataPlane and added the AssemblyInfo under Management/, so there is nothing to do about that one.